src/ControlKnob.vue

```diff
@@ -130,6 +132,7 @@ const downListener = (event: MouseEvent) => {
   mouseIsDown.value = true
   mouseMoved.value = false
   prevY = event.clientY
+  preventScrolling(event);
```

Comment:
Remove semicolons.

Feel free to add `.prettierrc` file to the root of the repo if it helps to guide your editor better. Here's the content for that file: 

```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": false,
  "singleQuote": true,
  "printWidth": 100
}
```